README.md

```diff
@@ -134,14 +134,15 @@ The generated `inspec.yml` file automatically points to the InSpec GCP resource
 name: my-profile
 title: My GCP InSpec Profile
 version: 0.1.0
-inspec_version: '>= 2.2.10'
+inspec_version: '>= 4.6.9'
 depends:
   - name: inspec-gcp
-    url: https://github.com/inspec/inspec-gcp/archive/master.tar.gz
+    url: https://github.com/inspec/inspec-gcp/archive/x.tar.gz
```

Comment:
We should figure out what our format is going to be and template this a bit better (eg `inspec-X.Y.tar.gz` or `compat` instead of `inspec`... arbitrary).... but just NOT saying "master" right now is more important than getting the format right. I'd even go so far as to ask that this get published immediately.